functions/tests/memberTagsHandler.test.js

```diff
@@ -183,6 +183,50 @@ describe("memberTagsHandler", () => {
     );
   });
 
+  it("should set tags from multidimensional nested config for new user", async () => {
```

Comment:
Once the other comments are resolved, please add an additional test for when keySelector is not provided but object format is.